### DISCUSSION TITLE
Text transparent with background image

### DISCUSSION THREAD

**Discussion by romainmartinez** [tailwindlabs/tailwindcss#2862] [Help] [answered]
Hello,

I try to reproduce this style with Tailwind:

![Screenshot from 2020-11-21 16-11-10](https://user-images.githubusercontent.com/17710617/99889725-ef971480-2c25-11eb-8e7b-29646b03deba.png)

Here is a [Tailwind play](https://play.tailwindcss.com/lst4DE7h4V) link with a minimal example.

I would like the text to be transparent and see the image through it.

Does anyone see how to do this? 
Thanks!

**Comment by chr-ge** [comment, marked as answer]
You can use `bg-clip-text` ([docs](https://tailwindcss.com/docs/background-clip#cropping-to-text)): https://play.tailwindcss.com/YGa2BXRcXT
Or, you can achieve this by using `mix-blend-mode`: https://play.tailwindcss.com/6xRcScjAON?file=css

**Reply by romainmartinez** [reply]
`mix-blend-mode` does the trick, thank you fellow Montrealer.

**Reply by simonswiss** [reply]
This was a great use of Tailwind Play to ask the question with maximum context 🎉

**Comment by MathGeniusJodie** [comment]
https://github.com/hacknug/tailwindcss-blend-mode
https://tailwindcss.com/docs/background-clip
